test/scsi_defs_test.cc

```diff
@@ -32,4 +36,25 @@ namespace {
     EXPECT_EQ(0b11, cb.vendorSpecific);
   }
 
+  /*
+    Tests the TestUnitReadyCommand sruct
+  */
+  TEST(TestUnitReadyCommandStruct, ShouldBePacked) {
+    scsi_defs::TestUnitReadyCommand command;
+    EXPECT_EQ(8, sizeof(command));
+  }
+
+  TEST(TestUnitReadyCommandStruct, ShouldReturnCorrectTestUnitReadyCommand) {
+    scsi_defs::OpCode opCode = scsi_defs::OpCode::kTestUnitReady;
+    uint64_t reserved = 8927;
+    uint64_t controlByte = 12;
+
+    std::bitset<48> cmdValue(static_cast<uint8_t>(opCode) + (reserved << 8) + (controlByte << 40));
```

Comment:
I believe that SCSI is big endian, and this seems to be assuming little endian (https://www.t10.org/ftp/t10/document.03/03-127r0.pdf). I'm not 100% on this so definitely let me know if I'm wrong!